pipelines/cemba/cemba_methylcseq/CEMBA.wdl

```diff
@@ -487,7 +487,7 @@ task CreateUnmappedBam {
     # disks should be set to 2.25 * input file size
     disks: "local-disk " + ceil(2.25 * (if input_size < 1 then 1 else input_size)) + " HDD"
     cpu: 1
-    memory: "3.5 GB"
+    memory: "3500 MB"
```

Comment:
Not MiB?

Reply:
Good point, I think I just got lazy. I'll fix this throughout